File: java/src/main/java/com/touzhijia/platform/service/PlatformException.java
package com.touzhijia.platform.service;

public class PlatformException extends Exception {
	private static final long serialVersionUID = -2125031025287319917L;

	private int code;
	
	public PlatformException(int code, String message) {
		super(message);
		this.setCode(code);
	}
	
	public int getCode() {
		return code;
	}

	public void setCode(int code) {
		this.code = code;
	}

}

File: c#/touzhijia/service/PlatformException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace touzhijia.service
{
    public class PlatformException : Exception
    {
        public int code { get; set; }

        public PlatformException(int code, String message):base(message)
        {
            this.code = code;
        }

    }
}
